src/v2/Apps/Consign/Routes/MarketingLanding/Components/CtaBanner.tsx

```diff
@@ -40,81 +58,81 @@ export const CtaBanner = () => {
     })
   }
 
+  const trackGetInTouchClick = () => {
+    trackEvent({
+      // @ts-ignore
+      action: "clickedGetInTouch",
+      // @ts-ignore
+      context_module: "StickyBanner",
+      context_page_owner_type: contextPageOwnerType,
+      label: "Get in Touch",
+      user_id: user?.id,
```

Comment:
**suggestion (non-blocking):** If the event schema has a field for `user_email` it could be convenient to include that so that we don't have to fetch the email of a user from its ID when we use this data.

Reply:
Added, also warned Audrey about the changes.